vaadin-dev-bundle/src/main/java/com/vaadin/devbundle/FakeAppConf.java

```diff
@@ -9,6 +9,14 @@
 @JsModule("@vaadin-component-factory/vcf-nav")
 @NpmPackage(value = "line-awesome", version = "1.3.0")
 @NpmPackage(value = "@vaadin-component-factory/vcf-nav", version = "1.0.6")
+@NpmPackage(value = "@vaadin/bundles", version = "%bundles.version%")
```

Comment:
Why do we need these?

Reply:
Just because they are missing in the dev bundle :)
To be honest I'm not much familiar with the `@vaadin/bundles`.
What I understand is that this package was included into dev bundle because there was a dependency to `vaadin` which imports it.
Now we change the structure of dependencies in platform and it's not available for bundle maker.
Why are the rest of dependencies are missing I have no idea.